apps/contribution-report/project.json

```diff
@@ -61,24 +61,46 @@
       }
     },
     "deploy": {
-      "executor": "@alexkeating/nx-skynet:deploy",
+      "executor": "@nrwl/workspace:run-commands",
+      "inputs": [
+        {
+          "env": "CLOUDFLARE_ACCOUNT_ID"
+        },
+        {
+          "env": "CLOUDFLARE_API_TOKEN"
+        }
+      ],
+      "dependsOn": [
+        {
+          "target": "build:production",
+          "projects": "self"
+        }
+      ],
       "options": {
-        "uploadDir": "dist/apps/contribution-report",
-        "registrySeedVarName": "CONTRIBUTION_REPORT_SKYNET_SEED",
-        "registryDataKey": "skylink.txt",
-        "portalUrl": "https://skynetfree.net",
-        "skynetJwtVarName": "SKYNET_JWT"
+        "command": "wrangler pages publish ./dist/apps/protocol-frontend --project-name=contribution-report",
```

Comment:
is it the FE or the contribution report that's being deployed? the command is inconsistent between prod and staging

Reply:
looks like this should be the contribution report